zebra-network/src/peer/handshake.rs

```diff
@@ -376,22 +384,44 @@ where
                 future::ready(Ok(msg))
```

Comment:
Open a follow-up for changing the codec without impacting any frames that are still being processed

Reply:
#1993